getting-started/proc-using-external-database-operator.adoc

```diff
@@ -0,0 +1,66 @@
+[id='proc-using-external-database-operator_{context}']
+//= Using an external database with the Operator
```

Comment:
this is an unsupported feature in 1.1. Please comment this out until we resolve https://issues.redhat.com/browse/RHIDP-453

Reply:
@Gerry-Forde Could you please remove or comment this? (This may be needed in a future RHDH version).